src/twisted/conch/ssh/keys.py

```diff
@@ -1202,11 +1205,33 @@ def _toString_OPENSSH(self, extra):
         """
         data = self.data()
         if self.isPublic():
+            if self.type() == 'EC':
+                if not extra:
+                    extra = b''
```

Comment:
In particular, I note that the RSA/DSA keys append `extra` with a space between the key ad the `extra`, but the code here does not. That's a cause for concern, especially when the behaviour of this code appears to be untested.